app/code/Magento/Store/Model/StoreManager.php

```diff
@@ -139,15 +144,21 @@ public function hasSingleStore()
     }
 
     /**
-     * {@inheritdoc}
+     * Check if system is run in the single store mode
+     *
+     * @return bool
      */
     public function isSingleStoreMode()
     {
         return $this->isSingleStoreModeEnabled() && $this->hasSingleStore();
     }
 
     /**
-     * {@inheritdoc}
+     * Retrieve application store object
```

Comment:
Please revert the comment and just remove curly braces `{}`. Please refer to the guide https://devdocs.magento.com/guides/v2.3/coding-standards/docblock-standard-general.html#inheritdoc